R/vdiagram.R

```diff
@@ -88,7 +88,8 @@ plotVdiagram <- function(data){
   vplot <- gsplot() %>%
     points(NA,NA, ylab='Stage, in feet', xlab='Shift, in feet') %>%
     addMinMax(getMinStage(data, required = TRUE), getMaxStage(data, required = TRUE), col = 'red', lwd = 3) %>%
-    grid(lty = "solid") %>%
+    grid(lty = "dotted", nx=25) %>%
+    grid(lty = "solid", nx=5, col="black") %>%
```

Comment:
I don't know that this is robust against different y and x ticks. The ticks may be 5 divisions, which would line up fine, but they also may be more or less which would clash. Should probably tie grid to axis as default in gsplot